main.go

```diff
@@ -184,6 +189,10 @@ func wrapMain() error {
 		// Base64 encode secret strings.
 		secrets[k] = base64.StdEncoding.EncodeToString([]byte(v.(string)))
 	}
+	for k, v := range vargs.SecretsBase64 {
+		// Don't base64 encode these secrets, they already are.
+		secrets[k] = v
```

Comment:
You probably also want to error when the variable shadows another variable set in `Secrets:` like this: https://github.com/NYTimes/drone-gke/pull/21/files#diff-7ddfb3e035b42cd70649cc33393fe32cR174